readthedocs/search/views.py

```diff
@@ -45,14 +46,18 @@ def elastic_search(request):
 
     if user_input.query:
         if user_input.type == 'project':
-            results = search_lib.search_project(
-                request, user_input.query, language=user_input.language)
+            project_search = ProjectDocument.faceted_search(query=user_input.query,
+                                                            language=user_input.language)
+            response = project_search.execute()
+            results = response.hits
+            facets = response.facets
```

Comment:
Is this used?

Reply:
Yes. Its used for showing facet (language) in project search results.